tests/rptest/chaos/redpanda_muservice.py

```diff
@@ -177,6 +177,9 @@ def _find_binary(self, service, name):
                             build_type, compiler, "dist/local/redpanda/bin",
                             name)
 
+        if not os.path.exists(path):
+            raise RuntimeError("Couldn't find binary %s: %s", name, path)
```

Comment:
You can also `assert os.path.exists(path), f"Couldn't find binary {name}: {path}"` since there really just isn't anything we can do about this.